dcm2bids/structure.py

```diff
@@ -219,9 +221,36 @@ def dstIntendedFor(self):
         return opj(
             self.participant.session,
             self.dataType,
-            self.participant.prefix + self.suffix,
+            self.set_order_entity_table(),
         )
 
+    def set_order_entity_table(self):
+        """
+        Return:
+            The destination filename formated following BIDS entity key table
+            https://bids-specification.readthedocs.io/en/latest/99-appendices/04-entity-table.html#appendix-iv-entity-table
```

Comment:
```suggestion
            The destination filename formatted following the v1.7.0 BIDS entity key table
            https://bids-specification.readthedocs.io/en/v1.7.0/99-appendices/04-entity-table.html
```

I think it is safer to mention the version of the spec than to rely on the _latest_ spec documentation.